src/core/logic.ml

```diff
@@ -61,6 +66,8 @@ end
    Range of de Bruijn indices in the dynamic scope.
 *)
 
+type lfTyp =  (* Used to classify the kind of LF object an LF type represents *)
+  | M | P
```

Comment:
These type constructors need some documentation. Does `M` stand for meta and `P` parameter?